tutorials/sap-cloud-alm-sap-build-2/sap-cloud-alm-sap-build-2.md

```diff
@@ -0,0 +1,213 @@
+---
+author_name: Xavier Dupeyrat
+author_profile: https://github.com/xavierdupeyrat
+keywords: tutorial
+auto_validation: true
+time: 60
+tags: [ tutorial>beginner, type>tutorial, software-product>sap-build, software-product>sap-build-apps--enterprise-edition ]
+primary_tag: software-product>sap-cloud-alm
+parser: v2
+---
+
+# SAP ALM API & Integration: Extension of SAP Cloud ALM with SAP Build (Part 2)
+<!-- description --> You will be developing a simple Build Apps app where you can interact with your projects and tasks inside a given destination linking to your SAP Cloud ALM API.
+
+
+## You will learn
+- How to build a simple **Build Apps** application to interact with SAP Cloud ALM entities
+
+## Prerequisites
+- You need to have the connectivity setup between SAP Build Apps and SAP Cloud ALM.
+- Check the previous [blog](https://developers.sap.com/tutorials/sap-cloud-alm-sap-build-process-automation-1.html
+ ) to setup your SAP Build and SAP Cloud ALM environments
+
+## Intro
+In the end of this blog you will finish with a Projects and a Tasks page as shown below:
+
+![intro](pic1.png)
+
+Clicking on a project it should send you to its Tasks page showing all tasks in this project.
+
+![intro](pic2.png)
+
+### Creating a data entity linking to the Projects endpoint.
+In the last tutorial you saw how to create a simple data entity linking to your main API. Here you will create a new one and link it to your projects endpoint.
+
+In the last tutorial you also enabled the SAP BTP Authentication necessary for the next steps.
+
+Navigate to the data tab at the top of the page. There you will be able to click on “Create Data Entity” and choose “Sap BTP Destination REST API integration”. After having done this a configuration tab for your data entity should open:
+
+![intro](pic3.png)
+
+Once on this page you need to choose the BTP destination name to link it to your Sap Cloud ALM API destination:
```

Comment:
Remove "Once on this page you need to" and start with "Choose" or "On this page, choose"